tools/install.sh

```diff
@@ -5,17 +5,49 @@ if [[ $(id -u) -ne 0 ]] ; then echo "Please run the installation script as root"
 SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
 CURRENT_PADIR="$(dirname "$SCRIPT_DIR")"
 PA_FOLDER_NAME="$(basename "$CURRENT_PADIR")"
+OLD_PADIR=
+OS=
+if which yum > /dev/null 2>&1; then
+   OS="RedHat"
```

Comment:
I would love to have support for _dnf_ which is the successor of _yum_ (yum is now deprecated on Fedora at least).